frontend/scripts/install_dev_env/install_macos.sh

```diff
@@ -41,9 +41,9 @@ printMessage "Setting up Flutter"
 
 # Get the current Flutter version
 FLUTTER_VERSION=$(flutter --version | grep -oE 'Flutter [^ ]+' | grep -oE '[^ ]+$')
-# Check if the current version is 3.10.1
-if [ "$FLUTTER_VERSION" = "3.10.1" ]; then
-   echo "Flutter version is already 3.10.1"
+# Check if the current version is 3.10.6
+if [ "$FLUTTER_VERSION" = "3.10.6" ]; then
```

Comment:
Please don't update the flutter version.

Reply:
I'm really sorry, I don't know how this got pushed. Will revert this